### ISSUE TITLE
Editable link labels

### ISSUE THREAD

**Issue by abhiravredox** [projectstorm/react-diagrams#317]
A lot of use cases for a diagram library involve the possibility of adding an editable text field to link. For example, https://live.yworks.com/demos/complete/bpmn/index.html and https://demo.bpmn.io/new
Currently, this feature isn't available in projectstorm. I would like to take up and implement this.

**Comment by nadeesh-garg** [comment]
Any updates on this feature?

**Comment by abhiravredox** [comment]
I haven't been assigned this issue so haven't been able to work on the same.

**Comment by dylanvorster** [comment]
You can do this now by implementing a custom Label

**Comment by smnodame** [comment]
@dylanvorster How to use a custom label? Do you have the example code ?
My usecase is to show remove button instead of label when the mouse is hover on the link

**Comment by Soarex16** [comment]
@dylanvorster, I have an example code, so If it passes tests, I can make a pr

**Comment by flieks** [comment]
@Soarex16 would you be willing to share some code snippet ;)

**Comment by Soarex16** [comment]
@flieks yes, I want to write example, forgot about it. Thanks for reminding :)

**Comment by Soarex16** [comment]
I think, this issue should be closed.
